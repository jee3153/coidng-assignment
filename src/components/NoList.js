import React, { useContext } from 'react'
import QueryContext from '../ContextAPI/QueryContext'

const NoList = () => {
  const queryHistory = useContext(QueryContext)

  return (
    <div className="NoList">
      <p>
        No result found with the keyword "
        <span className="NoList__query">{queryHistory}</span>
        "</p>
    </div>
  )
}

export default NoList