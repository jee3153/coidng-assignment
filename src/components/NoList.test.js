import React from 'react';
import renderer from 'react-test-renderer';

import NoList from './NoList';

describe('NoList Snapshot', () => {
  test('renders', () => {
    const component = renderer.create(
      <NoList />
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
