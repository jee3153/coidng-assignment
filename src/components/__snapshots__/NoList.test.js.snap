// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoList Snapshot renders 1`] = `
<div
  className="NoList"
>
  <p>
    No result found with the keyword "
    <span
      className="NoList__query"
    />
    "
  </p>
</div>
`;
